src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+        }
+    }
+
+    @Override
+    public Object onTraversalSuccess() {
+        printSummaryToStdErr();
+        writeConcordanceToBedFile();
+        writeSkippedEvalBedFile();
+        return "SUCCESS";
+    }
+
+    // =================================================================================================================
+
+    // TODO: 8/8/18 a nice feature to add is stratification by some features, e.g. genomic context, length, etc., and possibly even allowing users to specify the stratification key and levels
+    private void printSummaryToStdErr() {
+
+        // header
+        System.err.print(String.format("%-25s:  ", "type"));
+        for (final StructVarType type : StructVarType.values()) {System.err.print(String.format("%10s", type.name()));}
+        System.err.println();
+
+        final EnumMap<ConcordanceState, EnumMap<StructVarType, Long>> summaryByConcordanceThenBySvType =
+                classifiedVariants.stream()
+                        .collect(Collectors.groupingBy(EvalVariant::getConcordanceState,
+                                () -> new EnumMap<>(ConcordanceState.class),
+                                Collectors.groupingBy(classifiedEvalVariant ->
+                                                StructVarType.valueOf(classifiedEvalVariant.getEval().getAttributeAsString(GATKSVVCFConstants.SVTYPE, "")),
+                                        () -> new EnumMap<>(StructVarType.class),
+                                        Collectors.counting())));
+        summaryByConcordanceThenBySvType.forEach((concordanceState, countsBytype) -> {
+            System.err.print(String.format("%-25s:  ", concordanceState.name()));
+            final EnumMap<StructVarType, Long> zeroInitializedEnumMap = SVUtils.getZeroInitializedEnumMap(StructVarType.class);
+            countsBytype.forEach(zeroInitializedEnumMap::put);
+            zeroInitializedEnumMap.forEach((k, v) -> System.err.print(String.format("%10d", v)));
+            System.err.print("\n");
+        });
+
+        // for FN values
+        final EnumMap<StructVarType, Long> zeroInitializedEnumMap = SVUtils.getZeroInitializedEnumMap(StructVarType.class);
+        System.err.print(String.format("%-25s:  ", ConcordanceState.FALSE_NEGATIVE.name()));
+        Utils.stream(truthTree.iterator())
+                .flatMap(e -> e.getValue().stream())
+                .filter(classifiedTruth -> classifiedTruth.getConcordanceState().equals(ConcordanceState.FALSE_NEGATIVE))
+                .collect(Collectors.groupingBy(variant -> StructVarType.valueOf(variant.getTruth().getAttributeAsString(GATKSVVCFConstants.SVTYPE, "")),
+                        () -> new EnumMap<>(StructVarType.class), Collectors.counting()))
+                .forEach(zeroInitializedEnumMap::put);
+        zeroInitializedEnumMap.forEach((k,v) -> System.err.print(String.format("%10d", v)));
+
+        System.err.println();
+    }
+
+    private static final String NA_STRING = "NA";
```

Comment:
Constant need to be at the begining of the class code.

Reply:
done